src/app/study-sequence/study-sequence.component.html

```diff
@@ -25,7 +25,16 @@
       </app-sequence-sidebar>
     </div>
     <div class="check-show-answer">
```

Comment:
Can you space these buttons out more. This can be done by putting them in a grid or flex and designating the gap attribute.